app/visualisation_methods/sensitivity/plugin.rb

```diff
@@ -0,0 +1,212 @@
+require 'json'
+
+class Sensitivity
+  attr_accessor :experiment
+  attr_accessor :parameters
+
+  attr_reader :series_to_plot
+  attr_reader :parameters_names
+  attr_reader :sorted_parameters_names
+  attr_reader :name_of_output
```

Comment:
Już kiedyś mówiliśmy o zmiennych z "of" - wydaje mi się, że może być tutaj "output_name". Celem są krótsze nazwy zmiennych.